bigtable/examples/bigtable_samples.cc

```diff
@@ -28,15 +30,12 @@ namespace {
 const std::string MAGIC_ROW_KEY = "key-000009";
 
 //! [create table]
-void CreateTable(bigtable::TableAdmin& admin, std::string const& table_id) try {
+void CreateTable(bigtable::TableAdmin& admin, std::string const& table_id) {
   auto schema = admin.CreateTable(
       table_id, bigtable::TableConfig(
                     {{"fam", bigtable::GcRule::MaxNumVersions(10)},
                      {"foo", bigtable::GcRule::MaxAge(std::chrono::hours(72))}},
                     {}));
-} catch (std::exception const& ex) {
```

Comment:
This seems like a change in behavior; was that intentional?

Reply:
Yes.  The try/catch was not really needed and did not fit with the other examples.  Note that if an exception is raised the try/catch in main() will print out the error anyway.